### ISSUE TITLE
No error reported when calling unimplemented protocol method in explicit subclass

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#6880]
The [typing spec](https://typing.readthedocs.io/en/latest/spec/protocol.html#explicitly-declaring-implementation) indicates that the following situation should generate a type error, but pyright doesn't currently report it.

```python
from typing import Protocol

class ColorProto(Protocol):
    def draw(self) -> str:
        ...

class BadColor(ColorProto):
    def draw(self) -> str:
        return super().draw()  # Type error: no default implementation
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.345, which I just published. It will also be included in a future release of pylance.
